pyvista/examples/downloads.py

```diff
@@ -3612,22 +3636,24 @@ def download_lucy(load=True):  # pragma: no cover
     return _download_and_read('lucy.ply', load=load)
 
 
-def download_can(partial=False):  # pragma: no cover
+def download_can(partial=False, load=True):  # pragma: no cover
     """Download the can dataset mesh.
 
-    Original downloaded from Testing/Data/FileSeriesat from paraview.org. Used
+    Original downloaded from Testing/Data/FileSeries at from paraview.org. Used
```

Comment:
1. What does "at from paraview.org" mean? Typo of "at paraview.org"?
2. What does "Testing/Data/FileSeries at paraview.org" mean? I can't find that.

Reply:
I don't know.  I thought this was a typo, but it doesn't make sense in the changed form either.  kitware moved a lot of their testing data to a new site https://data.kitware.com/. So, this bread crumb might be stale.  Maybe we should just say "Original downloaded from Paraview."

Reply:
Yeah, I can't seem to find it on the new site either. I agree we should use something generic then. Should it be Paraview or Kitware then?

Reply:
This is my fault. I should have provided a direct link.

Just cite Kitware

```suggestion
    File obtained from `Kitware <https://www.kitware.com/>`_. Used
```